src/components/appBanner/AppBanner.js

```diff
@@ -1,19 +1,20 @@
-import './appBanner.scss';
-import avengers from '../../resources/img/Avengers.png';
+import avengersSrc from '../../resources/img/Avengers.png';
 import avengersLogo from '../../resources/img/Avengers_logo.png';
+import {Container} from "../app/App.style";
```

Comment:
Why do you import styles from another component ?

Reply:
It's bad idea. Later it can be problem in ui

Reply:
Container is global component which use in different places